src/translation/wininstaller/pt_PT.nsi

```diff
@@ -4,46 +4,46 @@ LangString DESKTOP_SET_SHORTCUT ${LANG_PORTUGUESE} \
     "Criar Atalho no Ambiente de Trabalho"
 
 LangString INVALID_FOLDER_MSG ${LANG_PORTUGUESE} \
-    "A pasta de destino já existe. Por favor, seleccione uma nova pasta de destino."
+    "A pasta de destino já existe. Por favor, selecione uma nova pasta de destino."
 
 LangString RUNNING_APP_MSG ${LANG_PORTUGUESE} \
-    "O ${APP_NAME} está em execução. Por favor feche-o e corra o instalador de novo."
+    "O ${APP_NAME} está em execução. Por favor, feche-o e execute o instalador novamente."
 
 LangString OLD_WRONG_VER_FOUND ${LANG_PORTUGUESE} \
-    "Detectámos uma versão antiga do ${APP_NAME} na sua pasta de Ficheiros de Programas 32 Bit. É recomendado removê-la antes de instalar uma nova versão do ${APP_NAME}. Deseja removê-la agora?"
+    "Nós detetamos uma versão antiga do ${APP_NAME} na sua pasta de Ficheiros de Programas 32 Bits. É altamente recomendado a sua remoção antes de instalar uma nova versão do ${APP_NAME}. Deseja removê-la agora?"
 
 LangString OLD_WRONG_VER_FOUND_CONFIRM ${LANG_PORTUGUESE} \
     "Se continuar sem remover a instalação antiga, a sua instalação poderá ser corrompida! Tem a certeza que não deseja remover a versão antiga?"
 
 LangString OLD_VER_REMOVE_FAILED ${LANG_PORTUGUESE} \
-     "Pedimos desculpa, mas não foi possível remover a versão antiga. Iremos tentar instalar a nova versão, mas pode também pressionar cancelar e tentar remover a versão antiga manualmente."
+     "As nossas desculpas, mas não foi possível remover a versão antiga. Nós iremos tentar instalar a nova versão, mas pode também clicar em Cancelar e tentar remover a versão antiga manualmente."
 
 LangString ASIO_DRIVER_HEADER ${LANG_PORTUGUESE} \
-     "Driver ASIO"
+     "Controlador ASIO"
 
 LangString ASIO_DRIVER_SUB ${LANG_PORTUGUESE} \
-     "Para usar o ${APP_NAME}, precisa de um driver ASIO"
+     "Para utilizar o ${APP_NAME}, precisa de um controlador ASIO"
 
 LangString ASIO_DRIVER_EXPLAIN ${LANG_PORTUGUESE} \
-     "O ${APP_NAME} precisa de um driver ASIO para fornecer áudio de baixa latência. Mais informações:"
+     "O ${APP_NAME} precisa de um controlador ASIO para fornecer áudio de baixa latência. Mais informação:"
 
 LangString ASIO_DRIVER_MORE_INFO ${LANG_PORTUGUESE} \
-     "Mais informações sobre o ASIO em jamulus.io"
+     "Mais informação sobre o ASIO em jamulus.io"
 
 LangString ASIO_DRIVER_MORE_INFO_URL ${LANG_PORTUGUESE} \
-     "https://jamulus.io/wiki/Installation-for-Windows#asio"
+     "https://jamulus.io/pt/wiki/Installation-for-Windows"
```

Comment:
Please do not remove #asio here. It should still match the anchor on the pt site.